### DISCUSSION TITLE
Meeting agenda 30th Nov

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#45] [Meeting Agenda]
The following is a draft agenda which I am assembling for the 30th Nov:

1. **Standard updates** - to include ISDA CDM recommendation
2. **Contribution process** - there are a number of contributions in the pipeline that are potentially 'no brainers', and a some others which don't feel quite ready yet. Rather than applying exactly the same process to each, should we take a different approach in some instances?
3. **Project Archiving** - following on from the [discussion in our previous meeting](https://github.com/finos/technical-steering-committee/blob/master/meetings/2022-10-26.md#project-archiving), this feels more like a project than a governance decision. Who wishes to be involved? And how do we run this?
4. **ISDA CDM** - we will be joined by various Association members in the last 30 mins of our call to discuss the ISDA CDM recommendations.
5. **OSFF New York** - finally, for those attending OSFF in New York, we'll discuss how and when we can meet up!

**Comment by jonfreedman** [comment]
A couple of the contributions to potentially discuss if we have time:

1. https://github.com/finos/community/issues/218
2. https://github.com/finos/community/issues/215

Some of the specific concerns raised by Neil and Tim are:

1. What it means to take a "mature" project from a member (JPMorgan in both cases) and potentially mark it as an incubating FINOS project
2. What it means to take a "mature" project and try and re-publish it under a FINOS namespace

Tim shared feedback around the on-boarding of Perspective, and user feedback around "apparent downgrade in stability". I am not sure how much we want to get drawn into the FINOS / project owner workflows, or discussing what feels like community management topics, I'd prefer to focus on providing our acceptance recommendation from a technical perspective and then move on.

**Reply by ColinEberhardt** [reply]
Hi Jon, 

Both of those contributions were my motivation for agenda item (2), where we talk about a process that more easily accommodates the current pipeline. My initial thinking is that for projects that are mature and very likely to meet our criteria, we don't assess as a group (as we did for Rosetta, Vuu, etc ...). Instead, I'd be happy to nominate a single TSC member to assess the contribution, write up their findings with respect to the [contribution principles](https://github.com/finos/technical-steering-committee/blob/master/contribution-principles.md), then share with the TSC for feedback ahead of publication. This would allow us to deal with the above much more swiftly, and give the TSC bandwidth for other matters.

**Reply by jonfreedman** [reply]
I like that idea, it's also a good way to avoid the TSC being a blocker and having to organise meetings for the whole group. I think everything we can do async we should try and make work until it doesn't work for some reason.

**Comment by jonfreedman** [comment]
Also occurred to me that given we'll be 5 months into the 6 month TSC pilot, and the final month is December we should start talking about the outcome of the pilot and our recommendation for 2023.

https://www.finos.org/community-spotlight/community-spotlight-tsc
"The FINOS [Governing Board](https://www.finos.org/governing-board) approved the formation in Q2 of a TSC on a trial basis (circa six months)."
